src/hooks/useAnalytics.ts

```diff
@@ -178,8 +178,8 @@ export const useAnalytics = () => {
     }
   }, [onboardingState, currentOnboardingStep]);
 
-  // AnalyticsEvent.OnboardingConnectWallet
-  // AnalyticsEvent.OnboardingDisconnectWallet
+  // AnalyticsEvent.ConnectWallet
+  // AnalyticsEvent.DisconnectWallet
```

Comment:
These AnalyticsEvent names didn't change right?

Reply:
nope, this comment has just been incorrect from the first commit 😆 . 
<img width="325" alt="Screenshot 2024-05-07 at 5 14 31 PM" src="https://github.com/dydxprotocol/v4-web/assets/37092291/d2831cd2-59f0-45bf-880b-819bf3887b5e">